frontend/public/locales/en/translation.json

```
@@ -635,7 +635,7 @@
     "international": "Topics related to Europe, international diplomacy, defense, terrorism and immigration",
     "education_culture": "Topics related to the education system, media, culture, social networks and the ethics of algorithms",
     "health": "Topics related to the health system, mental health, sports, end-of-life ethics, COVID-19 and vaccines",
-    "institutions_democracy": "Topics related to the organization of political life, democracy, territories and collectivities, public service and overseas",
+    "institutions_democracy": "Topics related to the organization of political life, democracy, territories and collectives, public service and overseas",
```

The "collectivités" is a french juridic form. I'm not sure it can be translated by "collectives". According to [Wikipedia](https://en.wikipedia.org/wiki/Territorial_collectivity) we could translate "collectivités territoriales" by "territorial authorities".

I'd suggest to replace "territories and collectivities" by "local authorities".

```suggestion
    "institutions_democracy": "Topics related to the organization of political life, democracy, local authorities, public service and overseas",
```